glambie/util/date_helpers.py

```diff
@@ -84,3 +87,52 @@ def get_year_timedelta(year: int) -> timedelta:
         - datetime(year=year, month=1, day=1)
     )
     return year_length
+
+
+def get_glaciological_years(glaciological_year_start: float, min_date: float, max_date: float, returntype="arrays"):
+    """
+    Returns start and end dates of glaciological years within a timespan (min_date to max_date).
+    Only full years within the min_date - max_date time period are included.
+
+    Parameters
+    ----------
+    glaciological_year_start : float
+        a floating point number of when the glaciological year should start (between 0 and 1), e.g. 0.75 for October
+    min_date : float
+        minimum date to be calculated for, in fractional years
+    max_date : float
+        maximum date to be calculated for, in fractional years
+    return_type : str, optional
+        type in which the result is returned. Current options are: 'arrays' and 'dataframe', by default 'arrays'
+
+    Returns
+    -------
+    Union[np.array, np.array] or pd.DataFrame, depending on specified return_type
```

Comment:
I think you mean Tuple here rather, than Union

Reply:
Done